plugins/woocommerce/tests/php/src/Internal/Caches/VersionStringsGeneratorTest.php

```
@@ -0,0 +1,395 @@
+		$this->sut->delete_version( '' );
+	}
+
+	/**
+	 * @testdox Negative TTL from filter is converted to 0.
+	 */
+	public function test_negative_ttl_is_converted_to_zero() {
+		// phpcs:disable Generic.CodeAnalysis.UnusedFunctionParameter
+
+		add_filter(
+			'woocommerce_version_string_ttl',
+			function ( $ttl, $id ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$id'. (undefined)

(UnusedFormalParameter)